Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -1570,9 +1570,12 @@ final class RecoveryTests: XCTestCase {
       class WrongInheritanceClause11️⃣(Int) {}
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: expected ':' to begin inheritance clause, Fix-It replacements: 30 - 31 = ': ', 34 - 35 = ''
-        DiagnosticSpec(message: "unexpected code '(Int)' in class")
-      ]
+        DiagnosticSpec(message: "expected ':' to begin inheritance clause",
+                       fixIts: ["replace '()' with ':'"])
```

Comment:
Not sure if this replacement is correct or it should be `replace '(Int)' with `: Int``

Reply:
I think it’s fine. Especially if the type is long, the current Fix-It message might be more readable.